### ISSUE TITLE
Remove a set of user accounts from Production database

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#6197]
Since we (by design!) don't allow deletion account deletion in Moped, only inactivation/archival, we will need to delete these directly in the production database. 

#### Accounts to be deleted
Alpha users with capital letters in email addresses (it turns out this interferes with SSO) ... _partial email addresses since Github stuff pulls so easily into Google search results_
- [x] Christopher.McC...
- [x] Eric.Bo...
- [x] Nathan.Wi...
- [x] Joel.Me...
- [x] laura.d... (even though it's lowercase, I think it needs to be redone)

Past team members
- [x] Leslie
- [x] JD
- [x] Mike

**Comment by sergiogcx** [comment]
@amenity As an update for this issue, @tillyw deleted the users with the exception of a couple few due to some Foreign Key constraints. It was late in the day, so I went ahead and carried it to the finish line. After removing the users from the DB, I also checked they were deleted from AWS, in both staging and development, but that only took 5 minutes.

Let @tillyw know if there is anything else that can be done for this issue, or feel free to reach out to me as well.
